portal-impl/test/integration/com/liferay/portal/dao/orm/hibernate/DynamicQueryTest.java

```diff
@@ -75,8 +82,42 @@ public void testCriterion() {
 	public void testInRestrictionCriterion() {
 		DynamicQuery dynamicQuery = ClassNameLocalServiceUtil.dynamicQuery();
 
-		List<Long> values = new ArrayList<>(
-			PropsValues.DATABASE_IN_MAX_PARAMETERS + 1);
+		List<Long> values = new ArrayList<>(2);
+
+		ClassName className1 = _allClassNames.get(1);
+		ClassName className2 = _allClassNames.get(2);
+
+		values.add(className1.getClassNameId());
+		values.add(className2.getClassNameId());
+
+		dynamicQuery.add(RestrictionsFactoryUtil.in("classNameId", values));
+
+		List<ClassName> classNames = ClassNameLocalServiceUtil.dynamicQuery(
+			dynamicQuery);
+
+		Assert.assertEquals(classNames.toString(), 2, classNames.size());
+		Assert.assertTrue(
+			classNames.toString(), classNames.contains(className1));
+		Assert.assertTrue(
+			classNames.toString(), classNames.contains(className2));
+	}
+
+	@Test
+	public void testInRestrictionCriterionWithMoreThanDatabaseInMaxParametersValue() {
+		DB db = DBManagerUtil.getDB();
+
+		DBType dbType = db.getDBType();
+
+		int databaseInMaxParameters = GetterUtil.getInteger(
+			PropsUtil.get(
+				PropsKeys.DATABASE_IN_MAX_PARAMETERS,
+				new Filter(dbType.getName())));
+
+		Assume.assumeTrue(databaseInMaxParameters > 0);
```

Comment:
@robertoDiaz this assume and https://github.com/shuyangzhou/liferay-portal/pull/6381/commits/1ba580c1066df5c549891b64b7d2b23673a004f5#diff-1fb7d9617dec4fc3e46bffea9f059656R61 are both wrong.
We provide this as a general feature, any customer can turn this on for their DB. 
By default, we only configure this for oracle, which means with these assume we don't test them on any other DBs, which just means we put this out without any tests for non-oracle customers. That is not acceptable.

You need to make these tests run on all dbs we have on CI. Remove those assumes and force set the in max size no matter what db you are using.